src/main/java/io/jenkins/docker/connector/DockerComputerSSHConnector.java

```diff
@@ -279,7 +279,7 @@ private InetSocketAddress getBindingForPort(DockerAPI api, InspectContainerRespo
         }
         String host = getExternalIP(api, ir, networkSettings, sshBindings);
 
-        return new InetSocketAddress(host, port);
+        return new InetSocketAddress("127.0.0.1", port);
```

Comment:
This will only work if Jenkins is running on the same host as the docker daemon.  It won't work in the general case where Jenkins is on a separate host from the docker hosts/swarms that it's talking to.